### PR TITLE
test(sort_package_xml): add tests

### PR DESCRIPTION
**Pull request by kenji-miyake**
Split from #32.

### CODE REVIEW

**Review by KeisukeShima — APPROVED** [review]
LGTM! :100:

**Comment by kenji-miyake** [comment]
I'll wait for comments from @isamu-takagi because he is the original author of these tests.

File: tests/test_sort_package_xml.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    # Format
+    return_code = sort_package_xml.main([str(tmp_file)])
+    print(tmp_file.read_text())
+    assert return_code == 1
+    assert tmp_file.read_text() == answer_file.read_text()
+
+    # Re-format
+    return_code = sort_package_xml.main([str(tmp_file)])
+    assert return_code == 0
```
Comment (isamu-takagi) [inline tests/test_sort_package_xml.py:33]: Check that the file has not been rewritten.
`assert tmp_file.read_text() == answer_file.read_text()`

File: tests/test_sort_package_xml.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    input_file = datadir / f"{case}.input.xml"
+    answer_file = datadir / f"{case}.answer.xml"
+
+    # Create a temporary file to prevent overwriting original files
+    tmp_file = Path(NamedTemporaryFile(mode="w", suffix=".xml").name)
+    shutil.copy(input_file, tmp_file)
+
+    # Format
+    return_code = sort_package_xml.main([str(tmp_file)])
+    print(tmp_file.read_text())
```
Comment (isamu-takagi) [inline tests/test_sort_package_xml.py:27]: Is this debug code?

Comment (kenji-miyake) [reply tests/test_sort_package_xml.py:27]: No. I thought that it's useful when the test has failed.

File: tests/test_sort_package_xml.py (hunk 1 above)
Comment (kenji-miyake) [reply tests/test_sort_package_xml.py:33]: Did you mean this? 71f0441086d5ae8e9fd275bd63d3cf0aff954620

Comment (isamu-takagi) [inline tests/test_sort_package_xml.py:33]:
```suggestion
    assert return_code == 0
    assert tmp_file.read_text() == answer_file.read_text()
```

**Review by isamu-takagi — APPROVED** [review]
LGTM

**Pull request merged**